web/blog/2022-06-16-vast-v2.1/README.md

```diff
@@ -0,0 +1,41 @@
+---
+draft: true
+title: VAST v2.1
+description: VAST v2.1 - TODO
```

Comment:
@mavam this and the other TODOs in this PR are all yours—feel free to just push to this PR directly.